src/main/java/core/db/AbstractTable.java

```diff
@@ -44,6 +44,17 @@ public AbstractTable(String tableName, ConnectionManager connectionManager) {
 		initColumns();
 	}
 
+	/**
+	 * Truncate string to maximum length
+	 * @param s String is truncated if it's length exceeds given limit
+	 * @param maxLength Length limit
+	 * @return Truncated string
+	 */
+	static String TruncateString(String s, int maxLength) {
```

Comment:
Would you mind using lowercase for the method name?  `truncateString`

Reply:
fixed - too much c++-programming in the last time....

Reply:
I suspected as much, there are telltale signs! ;-)
Thanks for taking the time to address this.